src/sdk/pynni/nni/msg_dispatcher.py

```diff
@@ -111,20 +111,8 @@ def handle_add_customized_trial(self, data):
 
     def handle_report_metric_data(self, data):
         if data['type'] == 'FINAL':
-            value = None
             id_ = data['parameter_id']
```

Comment:
I suggest we simply describe the desired schema for final metric data here.

Reply:
agreed